astropy/modeling/powerlaws.py

```diff
@@ -129,6 +129,142 @@ def fit_deriv(x, amplitude, x_break, alpha_1, alpha_2):
         return [d_amplitude, d_x_break, d_alpha_1, d_alpha_2]
 
 
+class SmoothlyBrokenPowerLaw1D(Fittable1DModel):
+    """One dimensional smoothly broken power law model.
+
+    Parameters
+    ----------
+    amplitude : float
+        Model amplitude at the break point
+    x_break : float
+        Break point
+    alpha_1 : float
+        Power law index for x << x_break
+    alpha_2 : float
+        Power law index for x >> x_break
+    smooth : float
+        Smoothness parameter
+
+    See Also
+    --------
+    BrokenPowerLaw1D
+
+    Notes ----- Model formula (with :math:`A` for ``amplitude`` and
```

Comment:
This part is generating Sphinx compilation errors on Travis. Did you try to build the HTML doc locally? If not, please do that and resolve all Sphinx errors and warnings associated with your changes. Also see how other models include their formulae and be consistent.

Posting a screenshot of the rendered doc for this portion would be nice, so reviewers can see what it is supposed to look like and check if the rendered formula is correct.